src/libs/actions/IOU.ts

```diff
@@ -6796,9 +6790,7 @@ function canApproveIOU(
 
     const managerID = iouReport?.managerID ?? -1;
     const isCurrentUserManager = managerID === userAccountID;
-    const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(chatReport);
-
-    const isOpenExpenseReport = isPolicyExpenseChat && ReportUtils.isOpenExpenseReport(iouReport);
+    const isOpenExpenseReport = ReportUtils.isOpenExpenseReport(iouReport);
```

Comment:
Previously, we checked the chat report to determine whether it was a policy expense chat, ensuring that such chats were tied to expense reports. Now, we don’t, which seems to have led to this [issue](https://github.com/Expensify/App/issues/52667).